code/raw_code/02_exploratory_analysis.R

```diff
@@ -1,212 +1,1077 @@
-  
-  library(sp)
-  library(maps)
-  library(maptools)
 
-  ## IMPORT DATA ################################################################################################################
-  raw_df <- read.csv(here("products", "NPU-Z", "SRG Survey Responses.csv"))
+  ######################### IMPORT DATA ######################### 
+  # Survey Responses
+  survey_df <- read.csv(here("data", "tidy_data", "pre_processed_survey_df.csv"))
+  survey_df$Stop_ID <- as.character(survey_df$Stop_ID)
+  
+  # Ridership Data
+  pre_covid_weekday <- read.xlsx(
+    here("data", "raw_data", "supplemental", "Average Weekday Saturday Sunday Stop Activity_November 1-15_2019-2020_MARTA Army.xlsx"),
```

Comment:
Would recommend placing the spreadsheet file name into a variable and use that. (5 other uses).